bin/full-node/src/run/network_service.rs

```diff
@@ -183,235 +233,83 @@ impl NetworkService {
-                                config,
-                                request,
-                            } => {
-                                tracing::debug!(%peer_id, "incoming-blocks-request");
-                                let mut _jaeger_span =
-                                    inner.jaeger_service.incoming_block_request_span(
-                                        &inner.local_peer_id,
-                                        &peer_id,
-                                        config.desired_count.get(),
-                                        if let (
-                                            1,
-                                            protocol::BlocksRequestConfigStart::Hash(block_hash),
-                                        ) = (config.desired_count.get(), &config.start)
-                                        {
-                                            Some(block_hash)
-                                        } else {
-                                            None
-                                        },
-                                    );
-
-                                let response =
-                                    blocks_request_response(&inner.databases[chain_index], config)
-                                        .await;
-                                request
-                                    .respond(match response {
-                                        Ok(b) => Some(b),
-                                        Err(error) => {
-                                            tracing::warn!(%error, "incoming-blocks-request-error");
-                                            None
-                                        }
-                                    })
-                                    .await;
-                            }
-                            service::Event::GrandpaCommitMessage {
-                                chain_index,
-                                message,
-                            } => {
-                                tracing::debug!(
-                                    %chain_index,
-                                    target_hash = %HashDisplay(message.decode().message.target_hash),
-                                    "grandpa-commit-message"
-                                );
-                            }
-                            service::Event::ProtocolError { peer_id, error } => {
-                                // TODO: handle properly?
-                                tracing::warn!(
-                                    %peer_id,
-                                    %error,
-                                    "protocol-error"
-                                );
-                            }
-                        }
-                    };
+        // Initialize the inner network service.
+        let inner = {
+            let (messages_from_connections_tx, messages_from_connections_rx) = mpsc::channel(64);
```

Comment:
Arbitrary. If it's too low, there might be more context switches than necessary because the tasks running the connections have to wait for the task running the network to do some work. If it's too high, messages will wait for a long time in queue, which adds latency and memory usage.